queries/nftSimpleListByAccount.graphql

```diff
@@ -0,0 +1,13 @@
+#import "./fragments/nft.graphql"
+#import "./fragments/nftDetails.graphql"
+
+query nftSimpleListByAccount($account: String!, $first: Int!) {
+  nftEntities(
+    where: { issuer_eq: $account, name_not_contains: "kanaria" }
```

Comment:
lol

Reply:
Just rewrote what was there.
Imo we do not have any kanaria NFTs

Reply:
> is it possible to "filter" directly on indexer side?

It's possible to do anything on indexer
and if not via Queries then I am 99% sure that we can write resolver for that